app/services/course_search_service.rb

```diff
@@ -39,9 +39,15 @@ def call
       outer_scope = outer_scope.descending_canonical_order
       outer_scope = outer_scope.select("provider.provider_name", "course.*")
     elsif sort_by_distance?
-      outer_scope = outer_scope.joins(sites_with_distance_from_origin)
-      outer_scope = outer_scope.select("course.*, distance")
-      outer_scope = outer_scope.order(:distance)
+      outer_scope = outer_scope.joins(courses_with_distance_from_origin)
+      boosted_distance_column = "
+      (CASE
+        WHEN provider.provider_type = 'O' THEN (distance - 10)
```

Comment:
This might be a bit clearer if we put these values in variables e.g.

```
university_provider_type = '0'
university_location_area_radius = 10
```